src/ng/compile.js

```diff
@@ -1221,6 +1228,11 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
       }
     };
 
+    function setSpecialAttr(element, attrName, value) {
+      specialAttrHolder.innerHTML = "<span " + attrName + "='" + (value || '').replace(QUOTE_REGEX, '&quot;') + "'>";
+      var attribute = specialAttrHolder.firstChild.attributes[0].cloneNode();
```

Comment:
[According to MDN][1], `cloneNode()` is supposed to be deprecated.

<sub>
It also says that, since Chrome v45, `Attr` no longer inherits from `Node`, but `cloneNode()` seems to still work on Chrome v46.
</sub>

[1]: https://developer.mozilla.org/en-US/docs/Web/API/Attr#Deprecated_properties_and_methods